ocp_resources/ingress.py

```diff
@@ -0,0 +1,5 @@
+from ocp_resources.resource import Resource
+
+
+class Ingress(Resource):
+    api_group = Resource.ApiGroup.CONFIG_OPENSHIFT_IO
```

Comment:
IIRC, this is also under the networking api group

Reply:
@sarahbx I guess you are right, here is an example I found
```apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: hello-openshift
spec:
  rules:
  - host: hello-openshift.yourcluster.example.com
    http:
      paths:
      - backend:
          # Forward to a Service called 'hello-openshift'
          service:
            name: hello-openshift
            port:
              number: 8080
        path: /
        pathType: Exact
```

Don't think it's namespaced resource, but it should be use networking api_group.

/unapprove

Reply:
We can get list/watch without the namespace, which returns an IngressList, but creation/deletion is namespaced
https://docs.openshift.com/container-platform/4.8/rest_api/network_apis/ingress-networking-k8s-io-v1.html

Reply:
BUT

ipdb> resource_definitions_dict["Ingress"]
{'namespaced': False, 'group': 'config.openshift.io'}


I'm confused.

Reply:
This is what got me confused but reading again the docs, yes - it should be namespaced.

Reply:
config api does also have access to ingress, but I think it should be separate from the namespaced ingress

https://docs.openshift.com/container-platform/4.8/rest_api/config_apis/ingress-config-openshift-io-v1.html

Reply:
what do you mean by that?
I could not fetch it by the networking. i need the specific ingress resource.
please clarify if you refer to the code i added.